lib/project_types/extension/messages/messages.rb

```diff
@@ -115,6 +115,8 @@ module Messages
         tunnel_already_running: "A tunnel running on another port has been detected. Close the tunnel and try again.",
       },
       tunnel: {
+        duplicate_session: "Another ngrok tunnel is currently running with your auth token, possibly on another machine. You must terminate that tunnel before opening a new one.\nAdditional information from ngrok logs: %s",
+        invalid_token: "The ngrok token you configured has expired. After generating a new token, you can update your local ngrok configuration using `shopify app tunnel auth <token>`\nAdditional information from ngrok logs: %s",
```

Comment:
Are you sure the token can expire? I can't find anything on the docs about that: https://ngrok.com/docs#getting-started-authtoken

Reply:
Good point! I think you can only rotate it manually. So perhaps something like, "The ngrok token you configured has been invalidated."